### DISCUSSION TITLE
Rewrites destination URL with colon

### DISCUSSION THREAD

**Discussion by ViLPy** [vercel/next.js#42420] [Help]
Hi,

Any ideas if it's possible to have a NextJS rewrite destination which contains colon? 
I need to proxy requests from, e.g, `/data/2022` to `https://some-service.com/param:2022/`, but when I try to do so with config from below it simply doesn't work and throws an error `Missing parameter name at xx`
```js
{
  source: '/data/:id',
  destination: 'https://some-service.com/param::id/'
}
```

Escaping `:` with `%3A` isn't supported on the end service.

**Comment by btakita** [comment]
I'm also running into this issue. Contabo's Object Storage public urls have a `:`, so it needs to be escaped in the  rewrite rule.

**Reply by btakita** [reply]
I fixed the destination url issue by using URL encoding (%3A).
